src/decider/agent/copyrightDeactivationClutterRemovalScript.py

```diff
@@ -0,0 +1,178 @@
+import spacy
+import pandas as pd
+import argparse
+import re
+import csv
+
+def preProcessing(file, clutter_flag):
+    with open(file, 'r') as f:
+        content = f.read()
+    df = pd.read_json(content, orient='index')
```

Comment:
You can pass the File-like object (`f`) as well instead of reading the string first to optimize memory consumption.